libbeat/common/jsontransform/jsonhelper.go

```diff
@@ -8,35 +8,36 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
-func WriteJSONKeys(event common.MapStr, keys map[string]interface{}, overwriteKeys bool, errorKey string) {
+// WritJSONKeys writes the json keys to the given event based on the overwriteKeys option
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function WriteJSONKeys should be of the form "WriteJSONKeys ..."